python/shotgun_model/shotgun_hierarchy_item.py

```diff
@@ -122,21 +122,19 @@ def target_entities(self):
         The dictionary also stores a ``type`` key whose value is the type of
         entity being targeted.
 
-        An example value returned by this method::
-
-            'target_entities': {
-                'additional_filter_presets': [
-                    {
-                        'path': '/Project/65/Asset/sg_asset_type/Character',
-                        'preset_name': 'NAV_ENTRIES',
-                        'seed': {
-                            'field': 'entity',
-                            'type': 'Version'
-                        }
-                    }
-                ],
-                'type': 'Version'
-            },
+        A common usage of this is to respond to selection changed signals
+        on a view showing a shotgun hierarchy. For example, the data returned
+        can be fed to a separate shotgun model view.
+
+        Example usage::
+
+            target_entities = selected_item.target_entities()
+
+            sg_model.load_data(
+                target_entities.get("type"),
```

Comment:
[nitpicky] The .get is overkill here, since all entity dicts will contain a type.